packages/lib-classifier/src/store/SubjectStore/SubjectStore.js

```diff
@@ -219,6 +228,34 @@ const SubjectStore = types
       }
     }
 
+    function * previous() {
+      const root = getRoot(self)
+      const workflow = tryReference(() => root.workflows.active)
+      const subject = tryReference(() => self.active)
+      const priority = subject?.priority
+      const subjectSetURL = `https://subject-set-search-api.zooniverse.org/subjects/${workflow.subjectSetId}.json`
+      const query = `priority__lt=${priority}&_size=1`
+      const mode = 'cors'
+      if (workflow?.hasIndexedSubjects) {
+        const response = yield fetch(`${subjectSetURL}?${query}`, { mode })
+        const { columns, rows } = yield response.json()
+        const subjectIDColumn = columns.indexOf('subject_id')
+        const [ row ] = rows
+        if (row) {
+          const previousSubjectID = row[subjectIDColumn]
+          const ids = previousSubjectID
+          const apiUrl = '/subjects/selection'
+          const params = {
+            ids,
+            workflow_id: workflow.id
+          }
+          const subjects = yield _fetchSubjects({ apiUrl , params })
+          self.append(subjects)
```

Comment:
We'll need a `prepend` method here, to sort the`resources` map by priority. Otherwise, subjects are presented in insertion order, which gets strange if you go back a couple of pages then forward again.